assets/styl/_modules/_form.styl

```diff
@@ -0,0 +1,59 @@
+// Forms widgets
+//
+// Colette provides styles for:
+// - Fields descriptions (with valid / invalid color indication)
+// - Valid / Invalid fields
+// - Also a special style for a flex parent with 3 children (label, infolink, input 100%)
+//
+// Note: you can also use Alerts components to handle success or errors messages.
+//
+// Weight: 2
+//
+// Markup: form.twig
+//
+// Styleguide: Components.Forms.Form
+.form
+    &-field-widget
+        // display a widget for form element that must be filled with text content
+        // e.g. input[text|password|email|url]
+
+        label
+            display block
+
+        ul
+            margin 0
+
+        //special style for a flex parent with 3 children (label, infolink, input 100%)
+        &-flex3
```

Comment:
It shouldn’t limited for 3 children because we need to be able to add.
Why not always use flex (by default all element to 100 % width)?
`.form-field-success` `.form-field-error` `.form-help-success` `.form-help-error` inside